oidc-server/app.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
```

Comment:
I see oidc-server but i don't a folder called ui?

Reply:
The source code for oidc ui moved under 'oidc-ui' folder.